src/root/POSRootChainManager.ts

```diff
@@ -9,6 +9,7 @@ import RootChain from './RootChain'
 
 const ERC20_TRANSFER_EVENT_SIG = '0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef'
 const ERC721_TRANSFER_EVENT_SIG = '0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef'
+const ERC721_TRANSFER_BATCH_EVENT_SIG = '0xf871896b17e9cb7a64941c62c188a4f5c621b86800e3d15452ece01ce56073df'
```

Comment:
This should be `ERC721_WITHDRAW_BATCH_EVENT_SIG`. It is not a transfer event in this case.